### PR TITLE
Update build.xml

### PR DESCRIPTION
**Pull request by robinshine**

### CODE REVIEW

File: componentA/build/build.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <javac srcdir="src" destdir="build/classes"/>
 	</target>
```
Comment (yjshen324) [inline componentA/build/build.xml]: yjshen324 - 1

File: componentA/build/build.xml [hunk 2]
```diff
@@ -26,12 +27,14 @@
 		</junit>
 	</target>
 	
+	
 	<target name="package" depends="compile, test">
```
Comment (yjshen324) [inline componentA/build/build.xml]: yjshen324 - 2

File: componentA/build/build.xml [hunk 3]
```diff
@@ -11,6 +11,7 @@
         <javac srcdir="src" destdir="build/classes"/>
 	</target>
 	
+	
 	<target name="test" depends="compile">
 		<mkdir dir="build/classes"/>
```
Comment (yujiangshen) [inline componentA/build/build.xml]: yujiangshen - 1

File: componentA/build/build.xml [hunk 4]
```diff
@@ -26,12 +27,14 @@
 		</junit>
 	</target>
 	
+	
 	<target name="package" depends="compile, test">
 		<echo message="start to package now"/>
 		<mkdir dir="package"/>
 		<jar basedir="build/classes" destfile="package/componentA-${buildVersion}.jar"/>
 	</target>
```
Comment (yujiangshen) [inline componentA/build/build.xml]: yujiangshen - 2

File: componentA/build/build.xml (hunk 1 above)
Comment (robinshine) [reply componentA/build/build.xml]: yjshen324 -1 -done

File: componentA/build/build.xml (hunk 2 above)
Comment (robinshine) [reply componentA/build/build.xml]: yjshen324 -2 -done

File: componentA/build/build.xml (hunk 1 above)
Comment (yujiangshen) [reply componentA/build/build.xml]: yjshen324 -1 -done -done

File: componentA/build/build.xml (hunk 2 above)
Comment (yujiangshen) [reply componentA/build/build.xml]: yjshen324 -2 -done -done

**Comment by robinshine** [comment]
Looks interesting

File: componentA/build/build.xml [hunk 5]
```diff
@@ -26,12 +27,14 @@
 		</junit>
 	</target>
 	
+	
 	<target name="package" depends="compile, test">
 		<echo message="start to package now"/>
 		<mkdir dir="package"/>
```
Comment (robinshine) [inline componentA/build/build.xml]: Doing somethhing

File: componentA/build/build.xml (hunk 1 above)
Comment (robinshine) [reply componentA/build/build.xml]: HAHAHA

**Pull request closed without merging**